### PR TITLE
Feature: sb 84 filters route

### PR DESCRIPTION
**Pull request by likeri29**
Added filters route to get all possible filters for products. 

[JIRA task](https://team-jfqe9wlknaql.atlassian.net/jira/software/c/projects/SB/boards/8?selectedIssue=SB-84)

### CODE REVIEW

File: src/modules/filters/filters.repository.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import { DataSource } from 'typeorm';
+
+@Injectable()
+export class FiltersRepository {
+  constructor(private readonly dataSource: DataSource) {}
+
+  async getAllFilters() {
```
Comment (lyudad) [inline src/modules/filters/filters.repository.ts]: 1.1.1

File: src/modules/filters/filters.repository.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    const [brands, categories, colors, materials, sizes, styles] =
+      await Promise.all([
+        this.dataSource.query('SELECT * FROM brands'),
```
Comment (lyudad) [inline src/modules/filters/filters.repository.ts]: why you use pure sql not typeorm functions?

File: src/modules/filters/filters.service.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@nestjs/common';
+import { FiltersRepository } from './filters.repository';
+
+@Injectable()
+export class FiltersService {
+  constructor(private readonly filtersRepository: FiltersRepository) {}
+
+  async getAllFilters() {
```
Comment (lyudad) [inline src/modules/filters/filters.service.ts]: 1.1.1

File: src/modules/filters/filters.repository.ts (hunk 1 above)
Comment (likeri29) [reply src/modules/filters/filters.repository.ts]: fixed

File: src/modules/filters/filters.service.ts (hunk 3 above)
Comment (likeri29) [reply src/modules/filters/filters.service.ts]: fixed

File: src/modules/filters/filters.repository.ts (hunk 2 above)
Comment (likeri29) [reply src/modules/filters/filters.repository.ts]: I used it because I didn't find a method how to use typeorm, but I refactored it.

**Review by VladManukian — APPROVED** [review]

File: src/modules/filters/filters.repository.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+import { DataSource } from 'typeorm';
+import {
+  BrandEntity,
+  CategoryEntity,
+  ColorEntity,
+  MaterialEntity,
+  SizeEntity,
+  StyleEntity,
+} from '../../entities';
```
Comment (ValentinaTenus) [inline src/modules/filters/filters.repository.ts:10]: consider using aliases

**Review by ValentinaTenus — APPROVED** [review]

**Review by jaroslawtt — APPROVED** [review]

**Pull request merged**